### DISCUSSION TITLE
How to hover any slide to pause in swiper

### DISCUSSION THREAD

**Discussion by nixjs** [nolimits4web/swiper#5843] [Q&A]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

https://codesandbox.io/s/marquee-with-swiperjs-tugmz8

### Bug description

I can not pause the slider when hover any element in swiper.

### Swiper version

8.2.2

### Platform/Target and Browser Versions

Any browsers

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR

**Comment by wjddlsy** [comment]
I think you should remove start/stop methods in mouse enter/leave callback. Because you already turn on pauseOnMouseEnter option when init swiper. If they are removed, it will be worked like below. 

![2022-07-04 17-23-48 2022-07-04 17_24_42](https://user-images.githubusercontent.com/22862991/177114090-ce6b4a44-16ab-4f5d-9fda-60b80c46cd55.gif)

However although after removing them, panels can not be stopped immediately. I don't know if this is the intended behavior.

**Comment by tresorama** [comment]
In `v9` with `react` `pauseOnMouseHover` does not always works. 
[Sandbox](https://stackblitz.com/edit/react-ts-5cjn3d?file=App.tsx).

**Comment by railsfactory-robin** [comment]
its taking time to stop, the slider will go all the way to center and stopping it, it should not be like this, it should stop immediately when user mouse hover

**Reply by tomferez** [reply]
@railsfactory-robin I found a working solution for this issue

**Reply by tresorama** [reply]
> @railsfactory-robin I found a working solution for this issue

Could you share it ?

**Comment by tomferez** [comment]
Edit: found a solution, just trigger this function on a mouseover event listener:

`const stopAnimation = () => {
                const translate = swiper.swiper.getTranslate();
                swiper.swiper.translateTo(translate, 0);
                swiper.swiper.autoplay.pause();
            };`

with pauseOnMouseEnter set to **true** passed to the Slider instance.

**Reply by railsfactory-robin** [reply]
@tomferez thanks for your update. its working as expected but only thing is while mouse leave its not resuming. i have tried something like below
```
const handleMouseEnter = () => {
      const translate = swiperInstance.getTranslate();
      swiperInstance.translateTo(translate, 0);
      swiperInstance.autoplay.pause();
    };

    const handleMouseLeave = () => {
      const translate = swiperInstance.getTranslate();
      swiperInstance.translateTo(translate, translate);
      swiperInstance.autoplay.start();
    };

**Reply by railsfactory-robin** [reply]
i found this solution

  
```
const handleMouseLeave = () => {
      const translate = swiperInstance.getTranslate();
      console.log(translate, "Mouse Leave - Current Translate");
      swiperInstance.translateTo(translate, 0);
      swiperInstance.autoplay.start();
      swiperInstance.update();
      console.log("Autoplay resumed and swiper updated");
    };

**Reply by railsfactory-robin** [reply]
still im facing an issue here then translate value is changed by the update method. its resetting to 0 so we can feel a jumping effect here. is there any way to avoid it?
 without update its not working, have tried with autoplay.start, resume etc..

**Reply by hqxc-sumail** [reply]
same how to solve ths problem

**Reply by hqxc-sumail** [reply]
> still im facing an issue here then translate value is changed by the update method. its resetting to 0 so we can feel a jumping effect here. is there any way to avoid it? without update its not working, have tried with autoplay.start, resume etc..

https://www.youtube.com/watch?v=-tWr7TIY3YQ watch thiss
